toolkit/smart-contracts/offchain/tests/legacy_governance_tx.rs

```diff
@@ -20,12 +21,12 @@ pub fn legacy_governance_init_transaction(
 				.to_bytes();
 		PCPlutusScript::from_wrapped_cbor(raw_scripts::MULTI_SIG_POLICY, Language::new_plutus_v2())
 			.unwrap()
-			.apply_uplc_data(uplc::PlutusData::Array(vec![
-				uplc::PlutusData::Array(vec![uplc::PlutusData::BoundedBytes(
-					governance_authorithy_key_hash.into(),
-				)]),
+			.apply_uplc_data(uplc::PlutusData::Array(MaybeIndefArray::Indef(vec![
```

Comment:
I am not sure, it is `MaybeIndefArray::Indef` or `MaybeIndefArray::Def`?

Reply:
Please use current version to encode it and both variants to see which encoding is the same

Reply:
Thanks, yea it looks like it is a correct version